lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -51,7 +51,8 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
   } = useContext(Config);
 
   // The current state of the error.
-  // One of "error-fetch", "error-authorizing", or "error-report-submission" or null
+  // One of "error-fetch", "error-authorizing", "error-report-submission",
+  // "error-unrecoverable" or null
   const [errorState, setErrorState] = useState(null);
```

Comment:
The `errorState` property has become overloaded to have two different meanings here:

- What activity failed? (`error-fetch`, `error-authorizing` and `error-report-submission` are all about this)
- What _kind_ of error occurred? (`error-uncoverable` is about this)

In the case of `error-unrecoverable` this is also duplicates information that is available in the `error` state.